src/TTrigger.cpp

```diff
@@ -350,15 +351,16 @@ bool TTrigger::match_perl(char* subject, const QString& toMatch, int regexNumber
     } else {
         unsigned char* tabptr;
         pcre_fullinfo(re.data(), nullptr, PCRE_INFO_NAMETABLE, &name_table);
-
         pcre_fullinfo(re.data(), nullptr, PCRE_INFO_NAMEENTRYSIZE, &name_entry_size);
-
         tabptr = name_table;
         for (i = 0; i < namecount; i++) {
+            int n = (tabptr[0] << 8) | tabptr[1];
+            auto name = QString::asprintf("%*s", name_entry_size - 3, tabptr + 2).trimmed();
```

Comment:
Qt documentation says:
> **Warning:** We do not recommend using `QString::asprintf()` in new Qt code. Instead, consider using `QTextStream` or `arg()`, both of which support Unicode strings seamlessly and are type-safe. Here's an example that uses `QTextStream`:
>> `QString result;`
>> `QTextStream(&result) << "pi = " << 3.14;`
>> `// result == "pi = 3.14"`
> For translations, especially if the strings contains more than one escape sequence, you should consider using the `arg()` function instead. This allows the order of the replacements to be controlled by the translator.
>
>This function was introduced in Qt 5.5.
>
> See also `arg()`.

Reply:
Yes! I've read it, and was pretty aware of that. `arg()` was not alternative in this case, not accepting range I guess... My understanding of this part is a bit limited.
It is based on:
https://github.com/vmg/pcre/blob/master/pcredemo.c

Reply:
I guess I've found solution not to use `asprintf`